website/docusaurus.config.js

```diff
@@ -183,7 +183,7 @@ const config = {
             dropdownItemsAfter: [
               ...Object.entries(ArchivedVersions).map(
                 ([versionName, versionUrl]) => ({
-                  to: versionUrl,
+                  href: versionUrl,
```

Comment:
this makes external links more visible on the dropdown:

![CleanShot 2023-08-17 at 19 42 15@2x](https://github.com/jestjs/jest/assets/749374/3fde8f17-92b1-47a0-8564-e5423973a552)